### PR TITLE
Downgrade aws k8s cni plugin version

### PR DESCRIPTION
**Pull request by artem-belov**
Signed-off-by: Artem Belov <artem.belov@xored.com>

## Description
Downgrade aws k8s cni plugin version; 
Enable aws interdomain tests

## Motivation and Context
Fixing interdomain issue on AWS cluster #1603 

## How Has This Been Tested?

- [x] Covered by existing integration testing
- [ ] Added integration testing to cover
- [ ] Tested locally
- [ ] Have not tested

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by nickolaev** [comment]
Please also enable the interdomain tests

**Comment by denis-tingajkin** [comment]
@nickolaev the tests are enabled by these changes https://github.com/networkservicemesh/networkservicemesh/pull/1737/files#diff-b4c9f5caf5b650fbd8fa7b9ebb075d2aR101-R118

File: scripts/aws/aws-k8s-cni.yaml [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+      tolerations:
+        - operator: Exists
+      containers:
+        - image: 602401143452.dkr.ecr.us-west-2.amazonaws.com/amazon-k8s-cni:v1.5.3
+          imagePullPolicy: Always
+          ports:
+            - containerPort: 61678
+              name: metrics
+          name: aws-node
+          #readinessProbe:
```
Comment (denis-tingajkin) [inline scripts/aws/aws-k8s-cni.yaml]: can be removed

Comment (artem-belov) [reply scripts/aws/aws-k8s-cni.yaml]: It is aws recommended yaml file, I did not change anything.

**Comment by artem-belov** [comment]
@nickolaev Denis is right, Interdomain tests are already enabled

**Pull request merged**